pkgs/desktops/plasma-5/kwin/default.nix

```diff
@@ -23,18 +25,25 @@ mkDerivation {
   buildInputs = [
     libepoxy lcms2 libICE libSM libcap libdrm libinput libxkbcommon mesa pipewire
     udev wayland xcb-util-cursor xwayland
+    libxcvt plasma-wayland-protocols wayland-protocols
 
     qtdeclarative qtmultimedia qtquickcontrols2 qtscript qtsensors
     qtvirtualkeyboard qtx11extras
 
     breeze-qt5 kactivities kcmutils kcompletion kconfig kconfigwidgets
     kcoreaddons kcrash kdeclarative kdecoration kglobalaccel ki18n kiconthemes
     kidletime kinit kio knewstuff knotifications kpackage krunner kscreenlocker
-    kservice kwayland kwayland-server kwidgetsaddons kwindowsystem kxmlgui
+    kservice kwayland kwidgetsaddons kwindowsystem kxmlgui
     plasma-framework libqaccessibilityclient
 
   ];
   outputs = [ "out" "dev" ];
+
+  postPatch = ''
+    find . -type f -name "*.py" \
+      -exec sed -i 's,#!/usr/bin/env python3,#!${python3}/bin/python,' {} \;
```

Comment:
We can't use patchShebangs, can we?

Reply:
Sure we can - I was suspecting something like this to exist, but could not find it. Will have a look at other usages of "patchShebangs" and adjust this.